apps/condo/domains/user/access/ChangeUserEmailService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice user.ChangeUserEmailService --type mutations`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+
+async function canChangeUserEmail ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return true
+}
```

Comment:
can service users call this endpoint?

Reply:
Yes, why not? Any authorized users can call